srml/assets/Cargo.toml

```diff
@@ -2,29 +2,34 @@
 name = "srml-assets"
 version = "0.1.0"
 authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
 
 [dependencies]
 hex-literal = "0.1.0"
 serde = { version = "1.0", default-features = false }
 parity-codec = { version = "3.0", default-features = false }
-parity-codec-derive = { version = "3.0", default-features = false }
+# Needed for the set of mock primitives used in our tests.
```

Comment:
If they are only required for tests, we should put them into `dev-dependencies`.